src/com/company/Main.java

```diff
@@ -0,0 +1,13 @@
+package com.company;
+
+import Services.Service;
+
+import java.text.ParseException;
+
+public class Main {
+
+    public static void main(String[] args) throws ParseException {
+        Service service = Service.getInstance();
```

Comment:
![image](https://user-images.githubusercontent.com/67580205/161223515-694c0018-2ad8-4b3f-b8be-2785812f98f0.png)
Meniul nu prea imi afiseaza corect, vad ca mi-a dublat afisarea cand am selectat 0

Reply:
![image](https://user-images.githubusercontent.com/67580205/161223980-40d66437-703c-4735-99d6-6f2bf23107ad.png)
si ramane hanging dupa ce am adaugat toate campurile pt computers

Reply:
![image](https://user-images.githubusercontent.com/67580205/161224882-f85d5ba7-4b95-4897-88b3-2dbab4d84bb7.png)
daca incerc sa fac get by id pe o lista care nu are inca niciun element, imi crapa

Reply:
![image](https://user-images.githubusercontent.com/67580205/161225856-1724419f-7977-468a-b498-f365e8f54532.png)
![image](https://user-images.githubusercontent.com/67580205/161225959-b082efc7-7cff-486a-9c0d-589adb745613.png)
update employee didn't update the employee